src/Output.cxx

```diff
@@ -1530,10 +1563,17 @@ void ASTVisitor::GetDeclAttributes(clang::Decl const* d,
 {
   for (auto const* a : d->specific_attrs<clang::AnnotateAttr>()) {
     attrs.push_back("annotate(" + a->getAnnotation().str() + ")");
+    this->OS << " annotation=\"" << this->EscapeMessage(a->getAnnotation().str()) << "\"";
```

Comment:
Let's add these new `annotation=` and `deprecation=` attributes only for CastXML-format output and not GCC_XML.